lib/ansible/plugins/lookup/cf_output.py

```diff
@@ -0,0 +1,115 @@
+# (c) 2015, Stephen Granger <stephen(at)trinimbus.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+import boto.ec2
+import boto.cloudformation
+import os
+import sys
+import time
+import pickle
+
+# region/stack/param
+class LookupModule(LookupBase):
+    def __init__(self, basedir=None, **kwargs):
+        self.basedir = basedir
+        self.cache_dir = os.path.join(os.environ['HOME'],'.stack_outputs')
```

Comment:
Safer to use os.getenv('HOME') because if 'HOME' doesn't exist, this will error out. With os.getenv, you can pass a default value as well if 'HOME' doesn't exist, otherwise it'll return None. In this particular case, you need to give it a default or the join will fail. So perhaps something like os.getenv('HOME', ''), which will return you an empty string if 'HOME' doesn't exist.